fm/echonest.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
Cool lets get #18 merged and I can merge it into this PR and move the `echonest` module into `thirdparty` and sort the migration sequencing